cpp/include/cuspatial/point_quadtree.hpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/types.hpp>
+
+#include <memory>
+
+namespace cuspatial {
+
+/**
+ * @brief Construct a quadtree structure from points.
+ *
+ * @see http://www.adms-conf.org/2019-camera-ready/zhang_adms19.pdf for details.
+ *
+ * @note `scale` is applied to x_min and y_min to convert x and y coodiantes into a Morton code in
```

Comment:
Maybe run a spell-checker?

```suggestion
 * @note `scale` is applied to x_min and y_min to convert x- and y-coordinates into a Morton code in
```